examples/CMakeLists.txt

```diff
@@ -49,3 +49,6 @@ foreach ( EXAMPLE ${EXAMPLES} )
     install( TARGETS ${TARGET_NAME} )
 
 endforeach()
+
+# install the julia controller example as well
+install( PROGRAMS simple_trixi_controller.jl TYPE BIN )
```

Comment:
Why move it to the `bin/` directory? Can you directly execute it? I understand your motivation, but it feels completely _off_ for me to put a file in `bin` that I cannot just execute with `./filename`, i.e., that has runnable content and is `chmod`ded with `+x`.

Reply:
Why not leave it in `share/`?

> The /usr/share hierarchy is for all read-only architecture independent data files.

They likely did not consider these kind of non-executable script files when they wrote the [FHS specs](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html#ftn.idm236091847456), but it's the closest I found.

Reply:
Ah, good reference. Thanks!

At the moment it is not installed at all. 
So where it exactly should it go?

- `share/libtrixi/`?
- `share/libtrixi/examples`?

libelexirs are installed to `share/libtrixi/LibTrixi.jl/examples`, but they are kind of a different thing.

Reply:
`share/libtrixi/examples`?

Reply:
Done!